telemetry-core/src/main/java/com/newrelic/telemetry/events/EventBuffer.java

```diff
@@ -21,9 +20,11 @@
  */
 public final class EventBuffer {
   private static final Logger logger = LoggerFactory.getLogger(EventBuffer.class);
-
+  private static final int maxNumberOfAttributes = 254;
+  private static final int maxAttributeNameLength = 255;
+  private static final int maxAttributeValueLength = 4096;
   private final Queue<Event> events = new ConcurrentLinkedQueue<>();
-
+  private final IngestWarnings ingestWarnings = new IngestWarnings();
```

Comment:
Can a new  instance  of IngestWarnings be injected through the EventBuffer constructor?